tasks/libs/common/utils.py

```diff
@@ -260,11 +252,8 @@ def get_build_flags(
     if python_home_3:
         ldflags += f"-X {REPO_PATH}/pkg/collector/python.pythonHome3={python_home_3} "
 
-    # If we're not building with both Python, we want to force the use of DefaultPython
-    if not has_both_python(python_runtimes):
-        ldflags += f"-X {REPO_PATH}/pkg/config/setup.ForceDefaultPython=true "
-
-    ldflags += f"-X {REPO_PATH}/pkg/config/setup.DefaultPython={get_default_python(python_runtimes)} "
+    ldflags += f"-X {REPO_PATH}/pkg/config/setup.ForceDefaultPython=true "
```

Comment:
I don't have the full context, but do we still need the `ForceDefaultPython` flag? 

From the comment in the `python/init.go` file

> // ForceDefaultPython has its value set to true at compile time if we should ignore
	// the Python version set in the configuration and use `DefaultPython` instead.
	// We use this to force Python 3 in the Agent 7 as it's the only one available.

Reply:
Most likely not, but I'd prefer to restrain changing this in this initial PR, I'll keep a note to iterate on these type of flags later on, but I fear this will lead to a lot of follow up deletions that will stack up quickly